tests/test_app.py

```diff
@@ -204,11 +204,12 @@ def setup_whole_audit(client, election_id, name, risk_limit, random_seed):
     # get the retrieval list for round 1
     rv = client.get('{}/jurisdiction/{}/1/retrieval-list'.format(url_prefix, jurisdiction_id))
     lines = rv.data.decode('utf-8').splitlines()
-    assert lines[0] == "Batch Name,Ballot Number,Storage Location,Tabulator,Ticket Number,Audit Board"
+    assert lines[0] == "Batch Name,Ballot Number,Storage Location,Tabulator,Ticket Numbers,Audit Board"
     assert len(lines) > 5
     assert 'attachment' in rv.headers['content-disposition']
 
-    num_ballots = len([line for line in lines[1:] if line!=""])
+    lines = csv.DictReader(io.StringIO(rv.data.decode('utf-8')))
```

Comment:
can we package this up as a utility function since we're reusing it in a couple of places?

Reply:
yeah, that's a nice idea.